core/src/main/java/org/apache/cxf/phase/PhaseInterceptorChain.java

```diff
@@ -325,6 +325,9 @@ public synchronized boolean doIntercept(Message message) {
                     }
                     pause();
                     throw ex;
+                } catch (AbortedInvocationException ex) {
+                    abort();
```

Comment:
@coheigea @ffang folks would appreciate if you could validate my implementation here. Introduction of `AbortedInvocationException` is a best effort attempt to terminate chunking transfer that may fail in the middle. Sadly, there are no standard means to deal with that (AFAIK).